includes/woocommerce-template.php

```diff
@@ -86,6 +86,8 @@ function dw_email_order_render_extra_fields($order, $sent_to_admin, $plain_text)
 		} catch(LicenseDateUnderTimeException $e) {
 			//This should never happen
 			$issueDrivingLicenseDateWarning = '<p style="color: red">Let op: TKM eerder dan 6 maanden</p>';
+		} catch(LicenseDateLongOverTimeException $e) {
```

Comment:
What does this have to do with the story?

Reply:
It is a bug